src/timeline/components/stateChronicles/TimelineNumericChronicle.vue

```diff
@@ -70,8 +72,8 @@ export default {
         activityStyle() {
             return {
                 'left': `${this.leftPosition}px`,
-                'min-height': `${ACTIVITY_HEIGHT}px`,
-                'max-height': `${ACTIVITY_HEIGHT}px`,
+                'min-height': `40px`,
```

Comment:
Same comment as above. Lets try to use constants where possible instead of magic numbers.